modules/rosa-troubleshoot-hcp-install.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * support/rosa-troubleshooting-installations-hcp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="rosa-troubleshoot-hcp-install_{context}"]
+= Troubleshooting {hcp-title} installation error codes
+
+The following table lists {hcp-title-first} installation error codes and what you can do to troubleshoot these errors.
+
+.{hcp-title} installation error codes
+[options="header",cols="3"]
+|===
+| Error code | Description | Resolution
+
+| OCM3999
+| Unknown error.
+| Check the cluster installation logs for more details, or delete this cluster and retry cluster installation. If this issue persists, contact support by logging in to the link:https://access.redhat.com/support/cases/#/case/list[*Customer Support* page].
+
+| OCM5001
+| {hcp-title} cluster provision has failed.
+| Check the cluster installation logs for more details, or delete this cluster and retry cluster installation. If this issue persists, contact support by logging in to the link:https://access.redhat.com/support/cases/#/case/list[*Customer Support* page].
+
+| OCM5002
+| The maximum resource tag size of 25 has been exceeded.
+| Check the cluster information to determine if you can remove any unnecessary tags you have specified and retry cluster installation.
+
+| OCM5003
+| Unable to establish an AWS client to provision the cluster.
+| You must create several role resources on your AWS account to create and manage a {hcp-title} cluster. Ensure that your provided AWS credentials are correct and retry cluster installation.
+
+For more information about {hcp-title} IAM role resources, see _ROSA IAM role resources_ in the _Additional resources_ section.
+
+| OCM5004
+| Unable to establish a cross-account AWS client to provision the cluster.
+| You must create several role resources on your AWS account to create and manage a {hcp-title} cluster. Ensure that your provided AWS credentials are correct and retry cluster installation.
+
+For more information about {hcp-title} IAM role resources, see _ROSA IAM role resources_ in the _Additional resources_ section.
+
+| OCM5005
+| Failed to retrieve AWS subnets defined for the cluster.
+| Review the provided subnet IDs and retry cluster installation.
+
+| OCM5006
+| You must configure at least one private AWS subnet for the cluster.
+| Review the provided subnet IDs and retry cluster installation.
+
+| OCM5007
+| Unable to create AWS STS prerequisites for the cluster.
+| Verify that account and operator roles have been created and are correct. For more information, see _AWS STS and ROSA with HCP explained_ in the _Additional resources_ section.
+
+| OCM5008
+| The provided cluster flavour is incorrect.
+| Verify that the provided name or ID is correct when you are using the `--flavour` argument and retry cluster creation.
```

Comment:
just a minor one, `--flavour` it's the CLI parameter, but we could hit this also when using another client. Should we keep it generic and say something like the "flavour parameter" or we usually have this form?